src/oge/helpers.py

```diff
@@ -257,22 +282,55 @@ def create_plant_ba_table(year):
         validate="m:1",
     )
     plant_ba.update({"ba_code_physical": plant_ba["ba_code_physical_map"]})
+    plant_ba.drop(columns="state")
 
     return plant_ba
 
 
-def identify_distribution_connected_plants(df, year, voltage_threshold_kv=60):
+def add_plant_nameplate_capacity(df: pd.DataFrame) -> pd.DataFrame:
+    """Adds nameplate capacity to input data frame.
+
+    Args:
+        df (pd.DataFrame): table with a 'plant_id_eia' column.
+
+    Returns:
+        pd.DataFrame: original data frame with additional 'capacity_mw' column.
     """
-    Identifies which plant_id_eia are "distribution grid connected" based on a voltage threshold.
+    generators_capacity = load_data.load_pudl_table(
+        "generators_eia860",
+        columns=["plant_id_eia", "generator_id", "report_date", "capacity_mw"],
+    )
+    plants_capacity = (
+        generators_capacity.query(
+            "report_date == @generators_capacity['report_date'].max()"
+        )
+        .groupby(["plant_id_eia"])["capacity_mw"]
+        .sum()
+        .round(2)
```

Comment:
Why is rounding necessary? Do we ever have data with more than 2 decimals?
We may want to keep the original level of detail

Reply:
Most values have only digit after the decimal point. The ones that don't have one digit after the decimal point followed by 9s, e.g., 3.5999999999999999

Reply:
ok thanks, this makes sense then